driver/tests/src/test/scala/org/virtuslab/ideprobe/ProbeDriverTest.scala

```diff
@@ -273,6 +264,41 @@ final class ProbeDriverTest extends IdeProbeFixture with Assertions with RobotPl
     )
   }
 
+  @Test
+  def runTestsInDifferentScopes(): Unit = {
+    fixture.copy(workspaceProvider = WorkspaceTemplate.FromResource("gradle-project")).run { intelliJ =>
+      intelliJ.probe.withRobot.openProject(intelliJ.workspace)
+      val moduleRef = ModuleRef("foo.test")
+
+      val moduleRunConfiguration = TestRunConfiguration.Module(moduleRef)
+      val runResult = intelliJ.probe.run(moduleRunConfiguration, None)
+      assert(runResult.suites.size == 2)
+
+      val directoryName = "java"
+      val directoryRunConfiguration = TestRunConfiguration.Directory(moduleRef, directoryName)
+      val directoryRunResult = intelliJ.probe.run(directoryRunConfiguration, None)
+      assert(directoryRunResult.suites.size == 2)
```

Comment:
I don't think this assert is macro based. We should use assertEquals(expected, actual) for better error messages